Jenkinsfile

```diff
@@ -3,7 +3,14 @@
 // https://github.com/feedhenry/fh-pipeline-library
 @Library('fh-pipeline-library') _
 
-fhBuildNode {
+node('nodejs4') {
```

Comment:
Not sure if I should leave this at `nodejs4`, or bump to `nodejs6`?

Reply:
Bump to Node 6 in my opinion

Reply:
@grdryn are nodejs4 nodes available? +1 on bumping to Node 6 also

Reply:
Bumped to node6 now.

@damienomurchu yes, nodejs4 pod templates are available that will create an agent pod in which to run the job if a nodejs4 node is requested. We still have some modules that request nodejs4 nodes, probably because that's still an option for cloud app deployment (I think it is anyway, is it?).